### PR TITLE
Drts4

### PR DESCRIPTION
**Pull request by rchauhan18**
Adding DRTs

### CODE REVIEW

**Review by singhashish-wpf — APPROVED** [review]

**Review by pchaurasia14 — APPROVED** [review]

File: src/Test/ElementServices/DRT/Parser/TestClasses/TestClasses.asmmeta [hunk 1]
```diff
@@ -0,0 +1,758 @@
+#include "versions.h"
```
Comment (Kuldeep-MS) [inline src/Test/ElementServices/DRT/Parser/TestClasses/TestClasses.asmmeta:1]: is this file required? If yes .. MIT license should be added

**Review by Kuldeep-MS — CHANGES_REQUESTED** [review]

File: src/Test/ElementServices/DRT/Parser/TestClasses/TestClasses.asmmeta (hunk 1 above)
Comment (rchauhan18) [reply src/Test/ElementServices/DRT/Parser/TestClasses/TestClasses.asmmeta]: Made the required changes

**Review by Kuldeep-MS — APPROVED** [review]

**Review by dipeshmsft — COMMENTED** [review]
Why did we remove the TestClasses.asmmeta.config file ??

**Review by dipeshmsft — APPROVED** [review]
